<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkyCovWidget</class>
 <widget class="QWidget" name="SkyCovWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Assign sky-coverages</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_manually">
           <property name="text">
            <string>manually</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_distance">
           <property name="text">
            <string>automatically</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_txt">
           <property name="text">
            <string>max twin telescope distance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_distance">
           <property name="suffix">
            <string> [m]</string>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_stations">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Station</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sky-Coverage ID</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>change default values (overrides settings below)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>time function</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>influence time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>influence distance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="comboBox_skyCoverageTimeType">
           <item>
            <property name="text">
             <string>cosine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>linear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>constant</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="influenceTimeSpinBox">
           <property name="suffix">
            <string> [s]</string>
           </property>
           <property name="maximum">
            <number>7200</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>3600</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="influenceDistanceDoubleSpinBox">
           <property name="suffix">
            <string> [deg]</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>5.000000000000000</double>
           </property>
           <property name="value">
            <double>30.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>distance function</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_skyCoverageDistanceType">
           <item>
            <property name="text">
             <string>cosine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>linear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>constant</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_sky_cov">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>#sta</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>influence distance</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>influence time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>distance function</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>time function</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>stations</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>show example plot</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_example">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_54">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Visual example&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; how the &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;sky-coverage&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; could look like. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;Red dots&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; mark observations.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt;Dark blue&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; means the &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;sky-coverage is fully saturated&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; in this area.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#000000;&quot;&gt;Yellow&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; means those are the &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;most preferred areas for a next scan&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;This is only a visual example to understand the settings selected at the left side!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;You can set the expected number of scans per hour and randomize new observations to get a clearer picture. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="title">
              <string>display</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_52">
              <item>
               <layout class="QFormLayout" name="formLayout_10">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_45">
                  <property name="text">
                   <string>scans per hour</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_30">
                  <item>
                   <widget class="QSpinBox" name="spinBox_skyCoverageTemplateRandomObservations">
                    <property name="toolTip">
                     <string>assumed number of scans per hour</string>
                    </property>
                    <property name="statusTip">
                     <string>assumed number of scans per hour</string>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="value">
                     <number>20</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_13">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_46">
                  <property name="text">
                   <string>marker size:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="horizontalSlider_skyCoverageMarkerSize">
                  <property name="toolTip">
                   <string>adjust marker size</string>
                  </property>
                  <property name="statusTip">
                   <string>adjust marker size</string>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>50</number>
                  </property>
                  <property name="value">
                   <number>13</number>
                  </property>
                  <property name="sliderPosition">
                   <number>13</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::NoTicks</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_43">
                  <property name="text">
                   <string>marker distance:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSlider" name="horizontalSlider_skyCoverageMarkerDistance">
                  <property name="toolTip">
                   <string>adjust marker distance</string>
                  </property>
                  <property name="statusTip">
                   <string>adjust marker distance</string>
                  </property>
                  <property name="minimum">
                   <number>10</number>
                  </property>
                  <property name="maximum">
                   <number>50</number>
                  </property>
                  <property name="singleStep">
                   <number>10</number>
                  </property>
                  <property name="pageStep">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>40</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_47">
                  <property name="text">
                   <string>color resolution</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QSlider" name="horizontalSlider_skyCoverageColorResultion">
                  <property name="toolTip">
                   <string>adjust color resolution</string>
                  </property>
                  <property name="statusTip">
                   <string>adjust color resolution</string>
                  </property>
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>64</number>
                  </property>
                  <property name="value">
                   <number>21</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_81">
                <item>
                 <spacer name="horizontalSpacer_18">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_skyCoverageTemplateRandom">
                  <property name="toolTip">
                   <string>randomize observations</string>
                  </property>
                  <property name="statusTip">
                   <string>randomize observations</string>
                  </property>
                  <property name="text">
                   <string>randomize</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_12">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupBox_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox_example</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_manually</sender>
   <signal>toggled(bool)</signal>
   <receiver>tableWidget_stations</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_manually</sender>
   <signal>toggled(bool)</signal>
   <receiver>tableWidget_sky_cov</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_manually</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_txt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_manually</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_distance</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
